products/llm_analytics/frontend/LLMAnalyticsScene.tsx

```diff
@@ -146,7 +146,7 @@ function LLMAnalyticsGenerations(): JSX.Element {
                                 return <></>
                             }
 
-                            const visualValue = truncateValue(value)
+                            const visualValue = truncateValue(value as string)
```

Comment:
I'd avoid type casting like this and rely on `unknown` in `truncateValue`. It's more type-safe and we've had bugs because of casting like this.